res/stomp_moveit.yaml

```diff
@@ -57,3 +57,8 @@ stomp_moveit:
     description: "Assumed time change between consecutive points - used for computing control costs",
     default_value: 0.1,
   }
+  path_marker_topic: {
```

Comment:
not a huge fan of topic parameters, but it might be required if we have multiple STOMP instances in the same node :man_shrugging:

Reply:
I am open to other solutions. In the long run, if we generalize this to be available to all planner plugins it could also be part of the motion plan request :thinking: